### ISSUE TITLE
Load Performance (Need profiler info)

### ISSUE THREAD

**Issue by essial** [OpenDiablo2/OpenDiablo2#117]
On my machine (tm) the game boots to the main menu in about 1.5 seconds, but for some people we've seen between 10 seconds to a full minute to load. I need profiler outputs attached so I can trace this speed issue down. It's hard for me to fix as I do not have this issue.

**Comment by davidsbond** [comment]
Hey @essial, I've used ebiten a bit so might be able to help out. Does this issue occur when people are running the precompiled binary? Or is it only occurring when doing `go run`? 

Having used ebiten in the past I've noticed that initial builds can take up to a minute due to the reliance on C bindings, after the initial build, subsequent runs/builds go much faster.

**Comment by essial** [comment]
We're talking about raw load times. I actually spent half of friday talking directly with hajimehoshi and we've determined that creating a bunch of tiny images causes ebiten to spam a lot of openGL calls. The temporary workaround is to manually construct an atlas, but that's created a new problem where some atlases may be too big. For this ticket I'm just collecting raw metrics as it has always ran fast for me, but I have really high PC specs and a stupidly fast hard drive.
